src/dune_engine/build_system.mli

```diff
@@ -167,6 +167,9 @@ end
 (** Build a request *)
 val build : 'a Action_builder.t -> 'a Memo.Build.t
 
+(** Similar to [build], but doesn't build the dependencies, only expand them *)
```

Comment:
Again, this comment doesn't seem to be true, since we do build dependencies in the `execute_action_stdout` case.